app/models/organization.rb

```diff
@@ -264,6 +265,20 @@ def earliest_reporting_year
     year
   end
 
+  def send_submission_notification(partner)
+    if email_notification_opt_in?
+      OrganizationMailer.request_submission_notification(organization: self, partner: partner).deliver_now
+      self.email_notification_opt_in = true
+      save
+      return :notification_sent
+    else
+      self.email_notification_opt_in = false
```

Comment:
These don't need to be changed - you just checked them earlier. You'd just be setting the values to whatever they're already set to.